pycsw/core/util.py

```diff
@@ -33,11 +33,15 @@
 import time
 import datetime
 import logging
-import urllib2
+try:
+    from urllib2 import Request, urlopen
+except:
+    from urllib.request import Request, urlopen
 from shapely.wkt import loads
 from owslib.util import http_post
 from pycsw.core.etree import etree
 
+from six import text_type
```

Comment:
Using six everywhere but `pavement.py`.  I do not know how Paver works, but six did not get imported from there.